src/main/java/com/prgrms/be02slack/common/exception/ExceptionController.java

```diff
@@ -2,19 +2,72 @@
 
 import javax.validation.ConstraintViolationException;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.annotation.Order;
 import org.springframework.http.HttpStatus;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.web.bind.MethodArgumentNotValidException;
 import org.springframework.web.bind.MissingPathVariableException;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
 
 @ControllerAdvice
 public class ExceptionController {
 
+  private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+  @Order(1)
```

Comment:
`@Order` 를 붙이면 어떻게 동작에 영향을주는지 궁금합니다! 혹시 참고하신 자료가 있나요?

Reply:
ExceptionHandler가 위에서부터 순차적으로 진행되게 하려는 의도였는데 찾아보니 @Order어노테이션은 클래스 레벨에서 사용하는거 같아요 제거하도록 하겠습ㄴ디ㅏ

Reply:
ExceptionHandler에도 @order를 붙여 순서를 정할수있습니다~ 
https://stackoverflow.com/questions/19498378/setting-precedence-of-multiple-controlleradvice-exceptionhandlers/19500823

Reply:
갓...

Reply:
ExceptionHandler 에 order를 사용한 답변의 댓글에 다음과 같이 exceptionhandlers의 order 어노테이션이 순서를 보장하지 않는다는 내용이 있는데 추가적으로 확인해봐야할 내용같습니다

No - not because its position in your source code - This is just your source code and as you know it will be compiled to class files afterwards - So the position of your exceptionhandlers in your code does not ensure any order